iceoryx_posh/source/roudi/port_pool.cpp

```diff
@@ -109,6 +115,21 @@ PortPool::addConditionVariableData(const ProcessName_t& process) noexcept
     }
 }
 
+cxx::expected<popo::EventVariableData*, PortPoolError>
+PortPool::addEventVariableData(const ProcessName_t& process) noexcept
```

Comment:
the name is confusing since a `EventVariableData` is returned not added, but so are the other method names

Reply:
I know, but I chose the name for consistency reasons.